test/functional/name_txnqueue.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+# Licensed under CC0 (Public domain)
+
+# Test that transaction queue works.
+
+from test_framework.names import NameTestFramework
+from test_framework.util import *
+
+class NameTransactionQueueTest(NameTestFramework):
+    def set_test_params(self):
+        self.setup_name_test ([[]] * 1)
+        self.setup_clean_chain = True
+
+    def run_test(self):
+        node = self.nodes[0] # alias
+        node.generate(200) # get out of IBD
+
+        self.log.info("Begin registering a name.")
+        # cribbed from name_immature_inputs.py by Daniel Kraft
+        addr = node.getnewaddress()
+        new = node.name_new("d/name", {"destAddress": addr})
+        newTx = node.getrawtransaction(new[0])
+        nameInd = self.rawtxOutputIndex(0, newTx, addr)
+
+        self.log.info("Mine that transaction.")
+        node.generate(1)
+
+        self.log.info("Create a NAME_FIRSTUPDATE with sequence=12...")
+        nameAmount = Decimal('0.01')
+        ins        = [{"txid": new[0], "vout": nameInd, "sequence": 13}] # 12 blocks
```

Comment:
Is my understanding correct that the sequence numbers here are just needed due to #426?  Or is there anything more to it?

Reply:
No, that's just because I want to test it like I envision it will be used. I could in theory use a transaction that's orphaned or something just as well.

Reply:
Yes sure, I mean, will sequence numbers be used "for real" because of #426, or is there another need for them even after #426 is fixed?  Just for my understanding.

Reply:
If #426 is going to be fixed, there is no need for them anymore, but I imagine they might continue to be used because of inertia.